docs/api.rst

```diff
@@ -485,6 +485,55 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+
+Querying token specific events
+------------------------------
+
+You can query for all new channels opened for a token by making a ``GET`` request to the following endpoint. ``GET /api/<version>/events/token/<token_address>``
```

Comment:
> As a user I would expect to be able to provide many token addresses if I saw plurar here.

That is not necessarily true for all users, let's leave personal impression aside for a moment and try to be consistent with ourselves and use collections in the URL:

> /api/<version>/channels/<channel_address>
> /api/<version>/tokens/<token_address>/partners